### PR TITLE
Add publishing of cookies received from browser

### PR DESCRIPTION
**Pull request by uarlouski**
Screen Shot 2019-11-11 at 8 21 51 PM

Header has its own background since it's sticky as we scroll down

### CODE REVIEW

File: vividus-plugin-web-app-to-rest-api/src/main/java/org/vividus/bdd/steps/integration/HttpRequestSteps.java [hunk 1]
```diff
@@ -16,23 +16,21 @@
...
 import org.vividus.http.HttpTestContext;
+import org.vividus.reporter.event.AttachmentPublisher;
 import org.vividus.ui.web.action.ICookieManager;
 
 public class HttpRequestSteps
 {
-    private static final Logger LOGGER = LoggerFactory.getLogger(HttpRequestSteps.class);
     @Inject private ICookieManager cookieManager;
     @Inject private HttpTestContext httpTestContext;
+    @Inject private AttachmentPublisher attachmentPublisher;
```
Comment (valfirst) [inline vividus-plugin-web-app-to-rest-api/src/main/java/org/vividus/bdd/steps/integration/HttpRequestSteps.java]: what about changing to constructor?

**Comment by valfirst** [comment]
Would you mind publishing a screenshot of sample attachment?

File: vividus-plugin-web-app-to-rest-api/src/main/java/org/vividus/bdd/steps/integration/HttpRequestSteps.java (hunk 1 above)
Comment (uarlouski) [reply vividus-plugin-web-app-to-rest-api/src/main/java/org/vividus/bdd/steps/integration/HttpRequestSteps.java]: are we going to completely migrate from field injection to constructor injection?

Comment (valfirst) [reply vividus-plugin-web-app-to-rest-api/src/main/java/org/vividus/bdd/steps/integration/HttpRequestSteps.java]: yes, I think

**Comment by ikalinin1** [comment]
Keep the log, to have an ability triage results without report?

**Comment by uarlouski** [comment]
> Keep the log, to have an ability triage results without report?
1) Ability triage results without report becomes useless when the number of threads grows
2) Do you propose to change logging level to info? otherwise it will not be used at all since end users are not supposed to look into the code and find logging, additionally we have to describe how to change logging level in vividus to make it able for end users to display that info
3) If so, the allure will contain html report and same values from logger, isn't it look redundant?

**Comment by ikalinin1** [comment]
> > Keep the log, to have an ability triage results without report?
> 
> 1. Ability triage results without report becomes useless when the number of threads grows
> 2. Do you propose to change logging level to info? otherwise it will not be used at all since end users are not supposed to look into the code and find logging, additionally we have to describe how to change logging level in vividus to make it able for end users to display that info
> 3. If so, the allure will contain html report and same values from logger, isn't it look redundant?

Debug information still could useful.

**Comment by uarlouski** [comment]
> > > Keep the log, to have an ability triage results without report?
> > 
> > 
> > 
> > 1. Ability triage results without report becomes useless when the number of threads grows
> > 2. Do you propose to change logging level to info? otherwise it will not be used at all since end users are not supposed to look into the code and find logging, additionally we have to describe how to change logging level in vividus to make it able for end users to display that info
> > 3. If so, the allure will contain html report and same values from logger, isn't it look redundant?
> 
> Debug information still could useful.

This debug info was never displayed and the end users never knew and will never know about these logs, so you purpose to change debug level to info/increase logging level?

**Comment by ikalinin1** [comment]
> > > > Keep the log, to have an ability triage results without report?
> > > 
> > > 
> > > 
> > > 1. Ability triage results without report becomes useless when the number of threads grows
> > > 2. Do you propose to change logging level to info? otherwise it will not be used at all since end users are not supposed to look into the code and find logging, additionally we have to describe how to change logging level in vividus to make it able for end users to display that info
> > > 3. If so, the allure will contain html report and same values from logger, isn't it look redundant?
> > 
> > 
> > Debug information still could useful.
> 
> This debug info was never displayed and the end users never knew and will never know about these logs, so you propose to change debug level to info/increase logging level and at the same time keep the data twice in logs and in HTML report?

Yep, also we should revise logging, since we have lots of debug level logging;
https://github.com/vividus-framework/vividus/search?p=1&q=%22LOGGER.debug%22&unscoped_q=%22LOGGER.debug%22

**Comment by valfirst** [comment]
Closes #150

**Pull request merged**